cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/CgmesExportTest.java

```diff
@@ -672,5 +672,52 @@ void testCanGeneratorControl() throws IOException {
         }
     }
 
+    @Test
+    void networkWithoutControlAreaInterchange() throws IOException {
+        Network network = DanglingLineNetworkFactory.create();
+        assertEquals(0, network.getAreaCount());
+
+        try (FileSystem fs = Jimfs.newFileSystem(Configuration.unix())) {
+            Path tmpDir = Files.createDirectory(fs.getPath("/temp"));
+            String baseName = "dl-net";
+
+            // Exporting with default behaviour, no default control area is written
+            Path tmpDirNoCA = tmpDir.resolve("network-no-ca");
+            Files.createDirectories(tmpDirNoCA);
+            network.write("CGMES", null, tmpDirNoCA.resolve(baseName));
+            Network networkNoCA = Network.read(new GenericReadOnlyDataSource(tmpDirNoCA, baseName));
+            assertEquals(0, networkNoCA.getAreaCount());
```

Comment:
I recommend to not export and reimport, but rather read the exported file.

```suggestion
            String eqFile = ConversionUtil.writeCgmesProfile(network, "EQ", tmpDir);
            assertFalse(eqFile.contains("cim:ControlArea"));
```